apps/.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+    "files.exclude": {
+        ".vscode/": true,
+        "build/": true,
+        "node_modules/": true,
+        "**/.git": true,
+        "**/.svn": true,
+        "**/.hg": true,
+        "**/CVS": true,
+        "**/.DS_Store": true
+    }
+}
```

Comment:
Did you mean to commit this vscode config file?

Reply:
No, I will fix that - thanks!